core/src/test/java/org/apache/iceberg/TestDataTableScan.java

```diff
@@ -130,4 +132,63 @@ public void testTableScanWithPlanExecutor() {
     Assert.assertEquals(2, Iterables.size(scan.planFiles()));
     Assert.assertTrue("Thread should be created in provided pool", planThreadsIndex.get() > 0);
   }
+
+  @Test
+  public void testReaddingPartitionField() throws Exception {
```

Comment:
typo: "readding" => "reading"

Reply:
That was supposed to be re-adding, I made it camelCase now to be less ambiguous.